Source/WebGPU/WebGPU/BindGroup.mm

```diff
@@ -68,18 +74,37 @@ static bool textureViewIsPresent(const WGPUBindGroupEntry& entry)
             bool bufferIsPresent = WebGPU::bufferIsPresent(entry);
             bool samplerIsPresent = WebGPU::samplerIsPresent(entry);
             bool textureViewIsPresent = WebGPU::textureViewIsPresent(entry);
+            if (static_cast<int>(bufferIsPresent) + static_cast<int>(samplerIsPresent) + static_cast<int>(textureViewIsPresent) != 1)
```

Comment:
I will drop the static_cast, which makes it `if (bufferIsPresent + samplerIsPresent + textureViewIsPresent != 1)`

since C++ guarantees promotion to int during the addition operation in this case